metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/converter/ReferenceSequenceConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.importer.converter;
+
+import org.springframework.core.convert.converter.Converter;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.ReferenceSequence;
+import uk.ac.ebi.ena.sra.xml.AssemblyType;
+
+import java.util.Arrays;
+
+public class ReferenceSequenceConverter implements Converter<AssemblyType, ReferenceSequence> {
+
+    @Override
+    public ReferenceSequence convert(AssemblyType assemblyType) {
+        return new ReferenceSequence(
+                assemblyType.getNAME(),
+                "NOT_SPECIFIED",  // ENA only specifies accession+version, not the patch
+                Arrays.asList(assemblyType.getAccession()),
+                ReferenceSequence.Type.ASSEMBLY  // ENA data model only has ASSEMBLY type
```

Comment:
I've looked through the entire `uk.ac.ebi.ena.sra.xml` package but could't find anything resembling these types. I also compiled a [spreadsheet](https://docs.google.com/spreadsheets/d/18IT7defkFXC0zkwhO0kQyGWOi1UbMmN9SjJq6B3yHfc/edit?usp=sharing) of all possible analysis types with accession examples, and in my experience they either contain a `ReferenceSequenceType`, which in turn links to an `ASSEMBLYType` (and _only_ to it), or they contain nothing at all. If you could provide me with a pointer, e. g. an accession example where you think the reference sequence is not an assembly, but instead a gene sequence or a transcriptome, I would be thankful.

This may be related to the refName/accession thing (see discussion [here](https://github.com/EBIvariation/metadata-ws/pull/50#discussion_r287836865)), because one of the analysis accessions returns something weird which is not an Assembly (https://www.ebi.ac.uk/ena/data/view/HAAA01000000&display=xml). May this be what we're looking for?